rest/route/github.go

```diff
@@ -360,7 +352,7 @@ func (gh *githubHookApi) handleCheckRunRerequested(ctx context.Context, event *g
 			"task":    checkRunTask,
 			"message": "finding refreshed task",
 		})
-		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(taskErr, "finding task '%s' for check run", checkRunTask))
+		catcher.Add(errors.Wrapf(taskErr, "finding task '%s' for check run", checkRunTask))
```

Comment:
if we couldn't get the refreshed task, do we still gain anything from calling updateCheckRun on it?

Reply:
yes, we would still like to update the output to "Task restarted" even if the details links to an older execution

Reply:
that makes sense. Can we rename it to "latestExecutionForTask" and add a comment here explaining that we would still like to update the output to "Task restarted" even if the details links to an older execution?

Also, where the comment says "Get the task again to ensure we have the latest execution." , can you please add to that explaining that the latest execution is for the link we display in github linking back to the task?